spec/hscode/config/bootstrap_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+describe Bootstrap::Config do
+  describe '#get' do
+    let(:root) { File.expand_path('../../../../', __FILE__) }
```

Comment:
I do think doing this would be nice. Rubocop complains about using a global variable. I think it favors using a module level constants eg `Hscode::APP_ROOT` or just `APP_ROOT` which we can include in any file it is needed.

What do you think?